packages/next/build/webpack/plugins/next-esm-plugin.ts

```diff
@@ -169,7 +169,7 @@ export default class NextEsmPlugin implements Plugin {
           }
 
           if (IS_PRESET_ENV.test(name)) {
-            presets.push(['@babel/preset-modules', { loose: true }])
+            presets.push([require('@babel/preset-modules'), { loose: true }])
```

Comment:
Please make this `require.resolve`, which I think we use everywhere else:
```suggestion
            presets.push([require.resolve('@babel/preset-modules'), { loose: true }])
```

Reply:
I'll change it if you want, but https://github.com/zeit/next.js/blob/dee28884042a7d5485166c961bd4a6d514260b5f/packages/next/build/babel/preset.ts is using `require`

Reply:
Interesting -- it shouldn't be, please change this! 🙏